src/mixins/filing-template-mixin.ts

```diff
@@ -289,10 +289,19 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           legalName: this.getBusinessLegalName
         },
         dissolution: {
+          dissolutionDate: this.getCurrentDate,
```

Comment:
This is YYYY-MM-DD in Pacific tz. Please verify that the API converts this to UTC.

PS In the case of `header.date` (filing date), it's not used for anything in the UI, and the API will overwrite it anyway (eg, "2021-11-15T22:22:02.470955+00:00").

Reply:
(I am not sure if the API keeps track of dissolution time. If so then it needs the correct tz offset in the db.)

Reply:
> This is YYYY-MM-DD in Pacific tz. Please verify that the API converts this to UTC.
> 
> PS In the case of `header.date` (filing date), it's not used for anything in the UI, and the API will overwrite it anyway (eg, "2021-11-15T22:22:02.470955+00:00").

I'm not sure what the API does with this date but according to the schema it is required, and in this format (YYYY-MM-DD)

Reply:
Where did you see this in the schema? I only see
```
                "dissolutionDate": {
                    "type": "string",
                    "format": "date"
                },
```
which doesn't tell me the format, really. 🤷‍♂️

Reply:
In the example data, is the only other reference.
As well as the ability to file with the address in that format.

Reply:
ok